praw/models/reddit/redditor.py

```diff
@@ -67,14 +67,14 @@ class Redditor(
                                          user-subreddit, provides a dictionary
                                          of additional attributes. See below.
     ``subreddit['banner_img']``          The URL of the user-subreddit banner.
-    ``subreddit['name']``                The fullname of the user-subreddit.
+    ``subreddit["name"]``                The fullname of the user-subreddit.
     ``subreddit['over_18']``             Whether or not the user-subreddit is
                                          NSFW.
     ``subreddit['public_description']``  The public description of the user-
                                          subreddit.
```

Comment:
It would be nice to just manually update these ones, since their neighbors have been fixed.